docs/src/library/code-blocks.md

```diff
@@ -18,6 +18,7 @@ function() {
     const another = [];
 }
 `` `
+
```

Comment:
added space after the markdown code block as we do when writing examples and it also stated in this page that -
>Make sure to leave space above and below the markdown code block to ensure it is rendered correctly.